### DISCUSSION TITLE
스터디 일정 공지 : 화요일 8시

### DISCUSSION THREAD

**Discussion by Emma-Hyejin** [wa-reureu-studyroom/toy-project-dailymusic#1] [Announcements]
스터디 일정 공지 입니다. 

23년 5월 30일 화요일 8시 디스코드 미팅방에서 만나요~ 
과제 목표는 80%~

**Comment by HyoJeong-Park** [comment]
오늘의 스터디 세줄 요약

1. 다같이 신나는 마음으로 PR 보냈다.
2. 효정쓰, 혜진쓰 파일 머지하고 정연쓰 파일 머지하려다가 충돌 생겼따.
3. git fetch, git pull, git reset HEAD^ 등 방법 총출동 했지만.. 했지만..

**Reply by HyoJeong-Park** [reply]
그래서 일단, 목요일(6월1일)까지 혜진쓰를 메인으로 방법을 찾아보기로 하고 figma로 넘어가서 메인, 커뮤니티, 플레이리스트 페이지 디자인을 했다구 한다~
